lib/bios/menus.robot

```diff
@@ -535,13 +536,36 @@ Remove Disk Password
     END
     Press Key N Times    1    ${SETUP_MENU_KEY}
 
+Send Reboot Command
+    Telnet.Write Bare    \x1bR\x1br\x1bR
+
 Tianocore Reset System
-    # EDK2 interprets Alt + Ctrl + Del on USB keyboards as reset combination.
-    # On serial console it is ESC R ESC r ESC R.
     IF    '${DUT_CONNECTION_METHOD}' == 'SSH'
         FAIL    SSH not supported for interfacing with TianoCore
     ELSE IF    '${DUT_CONNECTION_METHOD}' == 'Telnet'
-        Telnet.Write Bare    \x1bR\x1br\x1bR
+        FOR    ${i}    IN RANGE    0    5
```

Comment:
I do not like that we change it for Telnet only. It means now we will have two different methods of saving changes. One - more manual - for Telnet, the other one - for PiKVM. This sounds like asking for even more problems?=.